### ISSUE TITLE
Bug: let g:nvimux_new_window = 'term' does nothing

### ISSUE THREAD

**Issue by artem-nefedov** [Vigemus/nvimux#38]
I discovered that specifying `let g:nvimux_new_window = 'term'` in vimrc doesn't work, so I had to edit lua "vars" file instead. At the same time, `let g:nvimux_new_tab = 'term'` actually did work as expected (even before I edited "vars").

**Comment by hkupty** [comment]
Hi. Thanks a lot for your issue.

Previously, nvimux was written in VimL and configuring it through variables was the way to go. As neovim implemented the lua layer, given that lua is a much nicer language to work with (as it allows a cleaner separation of layers/modules, logic is simpler to describe), I've moved away from using vim variables as those would require more lua<->neovim interop, whereas setting up variables directly in lua pushes the boundaries to where it would be less-frequently invoked.

(All that preamble to say that) now, to have the same behavior, one should set it up using lua directly, like this:
```
lua << EOF
local nvimux = require('nvimux')

-- Nvimux configuration
nvimux.config.set_all{
  new_window = 'term',
}

nvimux.bootstrap()
EOF
``` 

I've checked the source code and there's still a proxy that tries to fallback to vim vars if the relevant var isn't found in lua, however, I believe that's going to be dropped whenever I set up to work on nvimux again..
